core/controllers/resources_test.py

```diff
@@ -526,10 +527,44 @@ def setUp(self):
             'flac': ['audio/flac']
         }
 
+        # The HANDLER_ARGS_SCHEMA dict in AudioUploadHandler uses the
+        # ACCEPTED_AUDIO_EXTENSIONS as a key value in the
+        # 'should_end_with' validator. The dict needs to be swapped
+        # seperately as it is a class variable.
```

Comment:
But it doesn't mean that we will swap the entire schema of the handler, right? (I mean these tests are written for expected schema of the handler and swapping the schema should not be done here, right?)

Reply:
Yes you are right. I think it is possible to swap only the 'should_end_with' validator instead of the entire schema, so other tests are not affected.

Reply:
Still, these tests are not testing the actual lines of code we are expecting, right?

Can you share more details on what happens when you try to swap ACCEPTED_AUDIO_EXTENSIONS? (Maybe share the code and error logs?)

Reply:
@DubeySandeep Yes sure.